Makefile

```diff
@@ -9,6 +9,7 @@ CHANGELOG_BASE ?= HEAD~
 CHANGELOG_TARGET ?= HEAD
 PROJECT := github.com/containers/libpod
 GIT_BASE_BRANCH ?= origin/master
+VERSION ?= $(shell git describe HEAD 2>/dev/null)
```

Comment:
Should we keep the `git fetch --tags` here?

Reply:
I'm not sure _when_ it is needed, but you can add it back if you want.

>        By default, any tag that points into the histories being fetched is
>        also fetched; the effect is to fetch tags that point at branches that
>        you are interested in. This default behavior can be changed by using
>        the --tags or --no-tags options or by configuring remote.<name>.tagOpt.

I had to remove it, since it was fetching **all** tags and all _their_ history...

`git clone --depth=1 --branch=2019.02.8 https://github.com/buildroot/buildroot `